overlord/hookstate/hookmgr.go

```diff
@@ -211,6 +211,12 @@ func (m *HookManager) doRunHook(task *state.Task, tomb *tomb.Tomb) error {
 		return fmt.Errorf("cannot read %q snap details: %v", hooksup.Snap, err)
 	}
 
+	// useful for integration tests
+	marker := fmt.Sprintf("/var/lib/snapd/%s.force-fail", info.Name())
```

Comment:
Hmm, not sure I'm super fond of this.

Reply:
I agree with zyga... I think we shouldn't be introducing logic to drive tests. Since we make copy of core.snap for this test, would it be possible to modify it to have a "broken" configure hook instead (i.e. exit 1)?